src/constants/constants.ts

```diff
@@ -30,7 +30,8 @@ export enum featureType {
 
 export enum featuresWithNoNames {
     auth = 'auth',
-    localization = 'localization'
+    localization = 'localization',
+    vuetify = 'vuetify'
 }
```

Comment:
idk what this is but i'm 100% sure it shouldn't be in this project. What's the point of declarative json files to define this stuff if we're also hard-coding? Declarative is the ONLY way to go.